src/frontend/components/DisplayContainer/ContainerLink.tsx

```diff
@@ -1,20 +1,20 @@
 import Link from "next/link";
 
 interface ContainerLinkProps {
-  href: string;
-  children?: string | JSX.Element | Array<JSX.Element | string>;
+    href: string;
+    children?: string | JSX.Element | Array<JSX.Element | string>;
 }
 export const ContainerLink = ({
-  href,
-  children,
-}: ContainerLinkProps): JSX.Element => {
-  return (
-    <Link href={href} target="_blank" rel="noopener noreferrer">
-      <a
-        className={"underline text-blue-600 decoration-blue-600 decoration-2 "}
-      >
-        {children ? children : href}
-      </a>
-    </Link>
-  );
+                                  href,
+                                  children,
+                              }: ContainerLinkProps): JSX.Element => {
+    return (
+        <Link href={href} target="_blank" rel="noopener noreferrer">
+            <a
+                className={"underline text-blue-600 decoration-blue-600 decoration-2 break-all"}
```

Comment:
Would you mind refactoring your PR just to include this change or opening a new one? This is a great suggestion, and I would love to work with you to get it on the codebase.